Headers/DebugServer2/Host/Windows/ExtraWrappers.h

```diff
@@ -19,6 +19,11 @@
 #define DS2_EXCEPTION_UNCAUGHT_COM 0x800706BA
 #define DS2_EXCEPTION_UNCAUGHT_USER 0xE06D7363
 #define DS2_EXCEPTION_UNCAUGHT_WINRT 0x40080201
+// This is a special exception code that is used by the Microsoft C++
+// runtime to notify the debugger that a thread has started or exited.
+// We don't care about this, so we just ignore it. Some context at
+// https://github.com/go-delve/delve/issues/1383.
```

Comment:
What do you think of moving this comment to Thread.cpp instead?  The information would be more readily available at the site of use which then also explains why it would immediately resume the process (for those unfamiliar with the exception).

Reply:
Done.